rustls-libssl/README.md

```diff
@@ -1 +1,50 @@
-# temporary
+<p align="center">
+  <img width="460" height="300" src="https://raw.githubusercontent.com/rustls/rustls/main/admin/rustls-logo-web.png">
+</p>
+
+rustls-libssl is a partial reimplementation of the OpenSSL 3 libssl ABI.
+It is written in rust and uses rustls.  It does not replace libcrypto:
+this is still required.
+
+[![rustls-libssl](https://github.com/rustls/rustls-openssl-compat/actions/workflows/libssl.yaml/badge.svg)](https://github.com/rustls/rustls-openssl-compat/actions/workflows/libssl.yaml)
+
+# Status
+
+This project is experimental.  We have aimed for the subset of libssl
+used by curl and nginx versions as shipped with Ubuntu 22.04 LTS.
+We have a [compatibility matrix](MATRIX.md) and
+[known issues](https://github.com/rustls/rustls-openssl-compat/issues).
+
+# Usage
+
+## Installation
+```shell
+$ curl https://github.com/rustls/rustls-openssl-compat/releases/latest/download/rustls-libssl_amd64.deb
```

Comment:
note: changed this to wget as it has better defaults for downloading a file. the curl equivalent would be:

```
$ curl --location --output rustls-libssl_amd64.deb https://github.com/rustls/rustls-openssl-compat/releases/latest/download/rustls-libssl_amd64.deb
```